tests/Doctrine/Performance/Mock/NonProxyLoadingEntityManager.php

```diff
@@ -271,9 +271,9 @@ public function hasFilters()
     /**
      * {@inheritDoc}
      */
-    public function find($className, $id)
+    public function find($className, $id, $lockMode = null, $lockVersion = null)
```

Comment:
This probably means that I don't really want to introduce a BC break in a patch release :/ What would be best way forward? Add `find` as `@method` in the interface?

Reply:
Yes. You cannot add additional parameters to the interface before 3.0.